scanner/src/test/kotlin/ScannerTest.kt

```diff
@@ -952,8 +952,11 @@ private class FakeProvenanceDownloader(val filename: String = "fake.txt") : Prov
  * validation.
  */
 private class FakePackageProvenanceResolver : PackageProvenanceResolver {
-    override fun resolveProvenance(pkg: Package, sourceCodeOriginPriority: List<SourceCodeOrigin>): KnownProvenance {
-        sourceCodeOriginPriority.forEach { sourceCodeOrigin ->
+    override fun resolveProvenance(
```

Comment:
There should also be some new test in this file to verify that the feature works as intended.